var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -350,6 +354,7 @@ def cmake_args(self):
 
         if "+rocm" in self.spec:
             options.append(self.define("CMAKE_CXX_COMPILER", self.spec["hip"].hipcc))
+            options.append(self.define("Kokkos_ENABLE_ROCTHRUST", True))
```

Comment:
It is enabled by default unless the user explicitly disable it so I am not sure it is worth bothering with it.

Reply:
i meant to address the review comment above.

Reply:
It is better to be explicit. And Kokkos can fail if `rocthrust` is not correctly found.